deepchem/feat/molecule_featurizers/sparse_matrix_one_hot_featurizer.py

```diff
@@ -0,0 +1,123 @@
+import logging
+from typing import List
+
+import numpy as np
+import scipy
+from deepchem.feat.base_classes import Featurizer
+from typing import Any, Iterable
+
+logger = logging.getLogger(__name__)
+codes = [
+    'A', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'K', 'L', 'M', 'N', 'P', 'Q', 'R',
+    'S', 'T', 'V', 'W', 'Y', 'X', 'Z', 'B', 'U', 'O'
+]
+
+class SparseMatrixOneHotFeaturizer(Featurizer):
+  """Encodes any arbitrary string as a one-hot array .
+
+  Standalone Usage:
```

Comment:
Add a paragraph explaining how the featurizer works and why and when want to use it.

Reply:
Should be 
```
Example
-------
```